### PR TITLE
[DOC] convert README from md to rst

### PR DESCRIPTION
**Pull request by PeerHerholz**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Summary
This PR converts the README from .md to .rst addressing #164. Except two links (API Documentation and PyCon 2020 Poster) nothing was changed. 

## Checklist

- [ ] All tests passing
- [ ] I have added tests to cover my changes
- [X] I have updated documentation (if necessary)
- [X] My code follows the code style of this project
(we are using `black`: you can `pip install pre-commit`,
run `pre-commit install` in the `pydra` directory
and `black` will be run automatically with each commit)

The `Singularity` related local tests are failing due to differences between linux and darwin. This is addressed in #293.  

## Acknowledgment
- [X] I acknowledge that this contribution will be available under the Apache 2 license.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -17,8 +17,8 @@ Feature list:
 5. Distributed execution, presently via ConcurrentFutures, SLURM,
 and Dask (this is an experimental implementation with limited testing)
 
-[[API Documentation](https://nipype.github.io/pydra/)]
-[[PyCon 2020 Poster](https://docs.google.com/presentation/d/10tS2I34rS0G9qz6v29qVd77OUydjP_FdBklrgAGmYSw/edit?usp=sharing)]
+[API Documentation](https://nipype.github.io/pydra/)
+[PyCon 2020 Poster](https://docs.google.com/presentation/d/10tS2I34rS0G9qz6v29qVd77OUydjP_FdBklrgAGmYSw/edit?usp=sharing)
```
Comment (effigies) [inline README.rst:21]: These aren't RST, are they?

```suggestion
[API Documentation](https://nipype.github.io/pydra/)
[PyCon 2020 Poster](https://docs.google.com/presentation/d/10tS2I34rS0G9qz6v29qVd77OUydjP_FdBklrgAGmYSw/edit?usp=sharing)
```

Comment (PeerHerholz) [reply README.rst]: Yeah, somehow my local `rst` preview and I got confused, resulting in committing wrong changes. The latest commit should fix that. Sorry!

**Comment by djarecka** [comment]
@PeerHerholz - could you please merge the master

is this ready?

**Comment by PeerHerholz** [comment]
Hi @djarecka,

sorry for the late reply. 
There are some merge conflicts, that however can be resolved easily. 
I can't merge, as I'm not part of the organization I think, thus one of you would have to do it.

**Pull request merged**